trade-finance-manager-ui/templates/utilisation-reports/partials/record-correction-history-html.njk

```diff
@@ -7,3 +7,7 @@
 {% endif %}
 
 <p class="govuk-body" data-cy="record-correction-automatic-notification-text">You will be automatically notified via email when the record correction comes through from the bank.</p>
+
+{{ recordCorrectionTable.render({
+  recordCorrectionRows: recordCorrectionDetails.recordCorrectionRows
+}) }}
```

Comment:
What does the page look like now if there are no record corrections? Should this not be inside an if so that we don't display the table if there aren't any?